proc/proc_windows.go

```diff
@@ -127,6 +127,20 @@ func Launch(cmd []string) (*Process, error) {
 		return nil, ProcessExitedError{Pid: dbp.Pid, Status: exitCode}
 	}
 
+	for _, thread := range dbp.Threads {
```

Comment:
If you want to start process with all its threads suspended, you can pass CREATE_SUSPENDED into CreateProcess.